ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -461,12 +460,16 @@ protected IngestionResult ingestFromStreamImpl(StreamSourceInfo streamSourceInfo
         }
     }
 
-    /*
-     * Set the policy that handles the logic over which data size would the client choose to directly use queued ingestion instead of trying streaming ingestion
-     * first.
-     */
-    public void setQueuingPolicy(ManagedStreamingQueuingPolicy queuingPolicy) {
-        this.queuingPolicy = queuingPolicy;
+    /**
+     * <p>setQueuingPolicyFactor</p>
+     * A factor used to tune the policy that handles the logic over which data size would the client choose to directly
+     * use queued ingestion instead of trying streaming ingestion first.
+     * Setting the factor will create a new {@link ManagedStreamingQueuingPolicy} with this factor, switching
+     * {@link ManagedStreamingQueuingPolicy.Default} which is created with no factor.
+     * @param factor - Default is 1.
+     **/
+    public void setQueuingPolicyFactor(double factor) {
```

Comment:
Update CHANGELOG.md with all the changes, mention this specifically as a breaking change